src/main/java/io/stargate/sgv2/jsonapi/exception/JsonApiException.java

```diff
@@ -25,6 +27,12 @@ public class JsonApiException extends RuntimeException implements Supplier<Comma
 
   private final Response.Status httpStatus;
 
+  private final String title;
+
+  private final String errorFamily;
+
+  private final String errorScope;
```

Comment:
Should perhaps be an enum instead of String?

Reply:
not a terrible idea, when we started this refactoring we were not sure what all of the scopes  would be . And we were happy to keep it simple as we will be throwing it away, we now have a better idea.  @hazel maybe refactor into an enum to reduce duplicate strings